inst/admiral-lookup-book.csv

```diff
@@ -416,11 +415,11 @@ ADVS,BDS,"PARAMCD=""HDCSDS""",Derive Head Circumference-for-age z-score,admiralp
-ADMH,OCCDS,SMQ02SCN,Derive scope (N) of Standardized MedDRA Query 2,admiral,derive_vars_query,https://pharmaverse.github.io/admiral/cran-release/reference/derive_vars_query.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_admh.R
-ADMH,OCCDS,CQ04NAM,Derive name of Customized Query 4,admiral,derive_vars_query,https://pharmaverse.github.io/admiral/cran-release/reference/derive_vars_query.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_admh.R
-ADMH,OCCDS,CQ04CD,Derive code of Customized Query 4,admiral,derive_vars_query,https://pharmaverse.github.io/admiral/cran-release/reference/derive_vars_query.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_admh.R
+ADMH,OCCDS,SMQ02NAM,SMQ 02 Name,admiral,derive_vars_query,https://pharmaverse.github.io/admiral/cran-release/reference/derive_vars_query.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_admh.R
+ADMH,OCCDS,SMQ02CD,SMQ 02 Code,admiral,derive_vars_query,https://pharmaverse.github.io/admiral/cran-release/reference/derive_vars_query.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_admh.R
+ADMH,OCCDS,SMQ02SC,SMQ 02 Scope,admiral,derive_vars_query,https://pharmaverse.github.io/admiral/cran-release/reference/derive_vars_query.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_admh.R
+ADMH,OCCDS,SMQ02SCN,SMQ 02 Scope (N),admiral,derive_vars_query,https://pharmaverse.github.io/admiral/cran-release/reference/derive_vars_query.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_admh.R
+ADMH,OCCDS,CQ04NAM,Customized Query 04 Name,admiral,derive_vars_query,https://pharmaverse.github.io/admiral/cran-release/reference/derive_vars_query.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_admh.R
+ADMH,OCCDS,CQ04CD,Customized Query 04 Code,admiral,derive_vars_query,https://pharmaverse.github.io/admiral/cran-release/reference/derive_vars_query.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_admh.R
 ADVS,BDS,"PARAMCD=""WSTCIR""",Derive Waist to Height Ratio,admiralmetabolic,derive_param_waisthgt,https://pharmaverse.github.io/admiralmetabolic/reference/derive_param_waisthgt.html,Template Example,https://github.com/pharmaverse/admiralmetabolic/blob/main/vignettes/advs.Rmd
 ADVS,BDS,"PARAMCD=""WSTHIP""",Derive Waist to Hip Ratio,admiralmetabolic,derive_param_waisthip,https://pharmaverse.github.io/admiralmetabolic/reference/derive_param_waisthip.html,Template Example,https://github.com/pharmaverse/admiralmetabolic/blob/main/vignettes/advs.Rmd
```

Comment:
It looks like we may have lost an entry, right? Do you know why that occured?

Reply:
Which 'entry' is missing??  I can grep the last 2.

Reply:
got it;   let me check now -
Please see file `inst/lookup.diff` , uses  diffdf::diffdf looks like 424 rows